crud/rooms.py

```diff
@@ -49,7 +49,7 @@ async def create_room(room: rooms_schema.RoomCreate):
     else: return {"result": "Error"}
```

Comment:
If you have an error you should raise an Exception with clear message

Reply:
ok